docs/docs/guides/sign_and_send_tx/local_wallet.md

```diff
@@ -0,0 +1,93 @@
+---
+sidebar_position: 0
+sidebar_label: 'Local wallet'
+---
+
+# Using Local Wallet
+
+The simplest way to sign and send transactions is using a local wallet:
+
+## Eth Transaction
+
+```ts
+// First step: initialize `web3` instance
+import { Web3 } from 'web3';
+const web3 = new Web3(/* PROVIDER*/);
+
+// Second step: add an account to wallet
+const privateKeyString = '0x1f953dc9b6437fb94fcafa5dabe3faa0c34315b954dd66f41bf53273339c6d26';
+const account = web3.eth.accounts.wallet.add(privateKeyString);
+
+// Make sure the account have enough eth on balance to send the transaction
+
+// Third step: sign and send the transaction
+// Magic happens behind sendTransaction. If a transaction is sent from an account that exists in a wallet, it will be automatically signed.
+try {
+	const receipt = await web3.eth.sendTransaction({
```

Comment:
I think it may also be helpful to show like in the [docs](https://docs.web3js.org/api/web3-eth/class/Web3Eth/#sendTransaction) `sendTransaction`  could return promievents and they can use listen onto  `sending`, `sent`, `transactionHash`... can be used too, what do u think?

Reply:
yep. you're right, thx

Reply:
I've added PromiEvent page